ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/blobs/versions/deneb/BlobSidecar.java

```diff
@@ -98,4 +99,12 @@ public KZGCommitment getKZGCommitment() {
   public KZGProof getKZGProof() {
     return getField7().getKZGProof();
   }
+
+  public String toLogString() {
+    return LogFormatter.formatBlobSidecar(
+        getIndex(),
+        getBlob().toBriefString(),
```

Comment:
I'm not sure what will be inside blobs usually, but I have a gut feeling that we could stick into some header on this slice. Probably worth to hash it. Though hash is not free

Reply:
ah... good point.. but the purpose is to identify them in explorers, if we hash it might be difficult. Let's see if this is enough, we can change it later.

Reply:
But I don't want to run hashing tbh, and I see that commitments are searchable as well..

https://blobscan.com/tx/0xbb49bc471c7a95363e975f64d7cee4e3671457042accab6371706eb17459fd2d

I think I'll print the full commitment for now.